runner/src/indexer/indexer.ts

```diff
@@ -205,21 +188,21 @@ export default class Indexer {
           setSpan.end();
         }
       },
-      debug: async (...log) => {
-        const debugLogEntry = LogEntry.systemDebug(log.join(' '), blockHeight);
-        return await this.writeLog(debugLogEntry, logEntries);
+      debug: (...log) => {
+        const debugLogEntry = LogEntry.systemDebug(log.join(' : '), blockHeight);
```

Comment:
```suggestion
        const debugLogEntry = LogEntry.userDebug(log.join(' : '), blockHeight);
```
Should we be using `user` here and below?

Reply:
Yep!